src/vs/base/browser/ui/list/listWidget.ts

```diff
@@ -428,6 +429,14 @@ class TypeLabelController<T> implements IDisposable {
 			if (typeof labelStr === 'undefined' || matchesPrefix(word, labelStr)) {
 				this.list.setFocus([index]);
 				this.list.reveal(index);
+
+				if (index === start) {
+					// Focus did not change with typing, re-announce element https://github.com/microsoft/vscode/issues/95961
+					const ariaLabel = this.list.options.accessibilityProvider ? this.list.options.accessibilityProvider.getAriaLabel(this.list.element(index)) : undefined;
```

Comment:
Join me in the future:

```suggestion
					const ariaLabel = this.list.options.accessibilityProvider?.getAriaLabel(this.list.element(index));
```